Dockerfile

```diff
@@ -1,8 +1,22 @@
-FROM ubuntu:jammy 
+FROM datajoint/pydev:python3.6
+
+RUN apt update && apt -y install mysql-client-5.7 netcat
+
 ENV TEAM="Seahawks"
-COPY ./README.md /home/ubuntu/files/README.md
-ADD https://raw.githubusercontent.com/datajoint/datajoint-python/master/CHANGELOG.md /home/ubuntu/files/CHANGELOG.md
-RUN echo HEY! > /home/ubuntu/files/file1.txt
-USER games:games
+
+RUN mkdir /seahawks
+
+WORKDIR /seahawks
+
+RUN git clone https://github.com/davidgodinez/seahawks-newsletter-pipeline.git /seahawks
```

Comment:
This might be tempting to do but try to understand logically what this is ultimately doing. Is there a simpler way to include these files into your image without going through GitHub?

Reply:
We will run a `COPY . /main` instead.